package.json

```diff
@@ -18,14 +18,17 @@
   "author": "Railsware Products Studio LLC",
   "license": "MIT",
   "dependencies": {
-    "axios": ">=0.27"
+    "axios": ">=0.27",
+    "nodemailer": "^6.9.4"
```

Comment:
I think we can add `nodemailer` as `peerDependencies`
- and maybe even mark it as optional using [peerDependenciesMeta](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#peerdependenciesmeta)
- as most of our users would use SDK-as-SDK and people who needs `nodemailer-transporter` would have their own nodemailer
- #risk `declare` trick would not work if we will have "our" nodemailer and users would install their own

Reply:
Actually, as I'm thinking further...
- What would happen if the user would try to use our SDK without `nodemailer` installed?
  - Would it be even possible, as we are exporting `MailtrapTransport` from the index 🤔 
- Let's double check during `alfa` testing 👍